components/Homepage/HomepageSplitHero.tsx

```diff
@@ -44,13 +44,13 @@ export const HomepageSplitHero = () => {
                 <img
                   src={getProcessedImage('https://a-us.storyblok.com/f/1005200/1390x1390/cb35b9488b/frame-96.jpg', '700x700')}
                   alt=""
-                  loading="eager"
+                  fetchPriority="high"
```

Comment:
Use fetchPriority=high for hero images. It has limited support so this is progressive enhancement
https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/fetchPriority